docs/book/src/quick-start.md

```diff
@@ -90,19 +101,19 @@ kubectl apply -f config/samples/
 
 ## Run It On the Cluster
 
-Build and push your image to the location specified by `IMG`:
+* Build and push your image to the location specified by `IMG`:
 
 ```bash
-make docker-build docker-push IMG=<some-registry>/controller
+make docker-build docker-push IMG=<some-registry>/<project-name>:tag
 ```
 
-Deploy the controller to the cluster with image specified by `IMG`:
+* Deploy the controller to the cluster with image specified by `IMG`:
 
 ```bash
-make deploy IMG=<some-registry>/controller
+make deploy IMG=<some-registry>/<project-name>:tag
 ```
 
-If you encounter RBAC errors, you may need to grant yourself cluster-admin
-privileges:
+**Note:** If you encounter RBAC errors, you may need to grant yourself cluster-admin
+privileges or be logged in as admin. For further information see an example over how to [create a sample user](https://github.com/kubernetes/dashboard/blob/master/docs/user/access-control/creating-sample-user.md#creating-sample-user).
```

Comment:
is there another link we can use?  Dashboard isn't the best place to link to considering its state.   Since this usually happens on GKE, maybe link here: https://cloud.google.com/kubernetes-engine/docs/how-to/role-based-access-control#iam-rolebinding-bootstrap

Reply:
Done 👍